code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -411,6 +412,9 @@ var/global/list/damage_icon_parts = list()
 	var/skeleton = (SKELETON in src.mutations)
 	if(skeleton)
 		race_icon = 'icons/mob/human_races/r_skeleton.dmi'
+	var/skelevox = (SKELEVOX in src.mutations)
+	if(skelevox)
+		race_icon = 'icons/mob/human_races/vox/r_voxboney.dmi'
```

Comment:
Even if it seems like that technically 'fixes' the problem, you're really just breaking the genetics system to do something that should be handled by the species datum anyway.